stories/Colors/Colors.story.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import Swatch from './Colors';
-import '../../src/css/tailwind.css';
+import '../../src/css/tailwind.src.css';
```

Comment:
the compiled tailwind css can be included globally in the Storybook `preview-head.html` that way the entire storybook is styled using our design system and you don't have to import it into every story

Reply:
I believe it needs to be required here so that webpack will recompile it. May be able to put it in the storybook config.